src/Benchmarks/Data/DapperDb.cs

```diff
@@ -58,6 +58,28 @@ public DapperDb(IRandom random, DbProviderFactory dbProviderFactory, IOptions<Ap
             return result;
         }
 
+        public async Task<World[]> LoadMultipleUpdatesRows(int count)
+        {
+            var result = await LoadMultipleQueriesRows(count);
+
+            using (var db = _dbProviderFactory.CreateConnection())
+            {
+                db.ConnectionString = _connectionString;
+                await db.OpenAsync();
+
+                for (int i = 0; i < count; i++)
```

Comment:
The fastest way is likely to use a TVP to stream the updated values to SQL Server. Statement concatenation (what we do in EF) may be faster when the number of updates is low.